### PR TITLE
Kiwix nav graph display layout

### LINKED ISSUE #2439: Adding layout attribute for each fragment in kiwix_nav_graph.xml
Having a UI description helps in understanding the nav-graph flow better. I want to add layout attribute ti each fragment in kiwix_nav_graph.xml

At present we have nav-graph something like :
image

It can be made something like:
layout_after_changes

Can I work on it?
More over we can recursively put recyclerview-item layout attribute in each fragment's recyclervoew for more UI improvements.

### PR DESCRIPTION
**Pull request by Dalakoti07**
Fixes #2439 .

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -37,15 +37,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```
Comment (macgills) [inline .idea/codeStyles/Project.xml:40]: can you revert this change?

Comment (Dalakoti07) [reply .idea/codeStyles/Project.xml:40]: @macgills you mean to revert this commit  https://github.com/kiwix/kiwix-android/pull/2443/commits/8126ee70164dae5916e8823759e2bdb5fb3e36fd

I actually took the code from here https://github.com/kiwix/kiwix-android/blob/develop/.idea/codeStyles/Project.xml

Comment (macgills) [reply .idea/codeStyles/Project.xml:40]: I don't want to see any changes to this file. This file is automatically generated from our settings and shared among all kiwix developers so we have consistent environments. This particular block is contributed by the kotlin plugin I believe so you may have it disabled or have an older version

Comment (macgills) [reply .idea/codeStyles/Project.xml:40]: I guess the point I was trying to get across is that manually editing this file is a fools errand as it just gets rewritten by your settings. The easiest thing is just not to add any changes to this file to a commit unless you intentionally set out to change everybody's settings

Comment (Dalakoti07) [reply .idea/codeStyles/Project.xml:40]: ok, I will see to it and commit the changes and make a new pull request.

Comment (Dalakoti07) [reply .idea/codeStyles/Project.xml]: @macgills I have made the changes please review it.

**Comment by macgills** [comment]
>Code changes for issue #2439 .

This does not link the issue so that when we merge this PR it closes the issue automatically, please edit the description to read `Fixes #2439 `

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (macgills) [reply .idea/codeStyles/Project.xml]: you can click the refresh symbol
image
to request additional reviews and I'll get a notification

Comment (macgills) [reply .idea/codeStyles/Project.xml]: unfortunately I am out of time today but I will most definitely review tomorrow

Comment (Dalakoti07) [reply .idea/codeStyles/Project.xml]: Sure, thank you.

**Review by macgills — APPROVED** [review]
Well done and congrats on your first contribution to kiwix

**Comment by Dalakoti07** [comment]
Thank you, @macgills for guidance, looking forward to contribute more.
